komga/src/main/kotlin/org/gotson/komga/interfaces/rest/SeriesController.kt

```diff
@@ -324,12 +328,52 @@ class SeriesController(
     @AuthenticationPrincipal principal: KomgaPrincipal,
     @PathVariable(name = "seriesId") seriesId: String
   ): ByteArray {
-    seriesRepository.getLibraryId(seriesId)?.let {
-      if (!principal.user.canAccessLibrary(it)) throw ResponseStatusException(HttpStatus.FORBIDDEN)
-    } ?: throw ResponseStatusException(HttpStatus.NOT_FOUND)
+    val series = seriesRepository.getLibraryId(seriesId) ?: throw ResponseStatusException(HttpStatus.NOT_FOUND)
```

Comment:
Why the change of form if the content is the same?

Reply:
It's because I found it hard to follow, and it goes for a lot of places where an elvis operator is present at the end of a `let` statement, even worse when there is a lot of content inside of the `let`

Reply:
one line is not a lot of content :) If you don't touch this part of the code, don't refactor it in this PR 🙏🏻